README.md

```diff
@@ -39,9 +41,56 @@ This code is distributed under the [Apache 2 license](LICENSE).
 
 > The code statically links to code distributed under the LGPL license.
 
+## Table of Contents
+
+- [github.com/kaleido-io/ethconnect](#githubcomkaleido-ioethconnect)
+  - [About kaleido-io/ethconnect](#about-kaleido-ioethconnect)
```

Comment:
I'm a huge fan of ToC so I really love this. I think removing the material that comes before the ToC would make the experience a little cleaner.